src/renderer/store/selectors/contacts.ts

```diff
@@ -188,176 +170,184 @@ const currentChannel = createSelector(
         }
-      }
-    ).filter((item) => item !== undefined)
+      })
+      .filter(item => item !== undefined)
   }
 )
 
-export const directMessages = address => createSelector(
-  messagesOfChannelWithUserInfo,
-  channelOwner(address),
-  (messagesWithUserInfo, channelOwner): IDirectMessage => {
-    const messagesObjectsArray = messagesWithUserInfo.map((message) => {
-      const newMessage = {
-        ...message.message,
-        createdAt: Math.floor(message.message.createdAt),
-        sender: {
-          username: message.userInfo ? message.userInfo.username : 'unNamed',
-          replyTo: ''
+export const directMessages = address =>
+  createSelector(
+    messagesOfChannelWithUserInfo,
+    channelOwner(address),
+    (messagesWithUserInfo, channelOwner): IDirectMessage => {
+      const messagesObjectsArray = messagesWithUserInfo.map(message => {
+        const newMessage = {
+          ...message.message,
+          createdAt: Math.floor(message.message.createdAt),
+          sender: {
+            username: message.userInfo ? message.userInfo.username : 'unNamed',
+            replyTo: ''
+          }
         }
-      }
-      return newMessage
-    })
-    const sortedMessages = messagesObjectsArray.sort((a, b) => {
-      return b.createdAt - a.createdAt
-    })
-      .map(message => message)
+        return newMessage
+      })
+      const sortedMessages = messagesObjectsArray
+        .sort((a, b) => {
+          return b.createdAt - a.createdAt
+        })
+        .map(message => message)
 
-    const messages = sortedMessages
+      const messages = sortedMessages
 
-    const channelModerators = []
-    const messsagesToRemove: DisplayableMessage[] = []
-    const blockedUsers = []
-    let visibleMessages: DisplayableMessage[] = []
-    for (const msg of messages.reverse()) {
-      switch (msg.type) {
-        case MessageType.AD:
-          if (!blockedUsers.includes(msg.pubKey)) {
-            visibleMessages.push(msg)
-          }
-          break
-        case MessageType.BASIC:
-          if (!blockedUsers.includes(msg.pubKey)) {
-            visibleMessages.push(msg)
-          }
-          break
-        case MessageType.TRANSFER:
-          if (!blockedUsers.includes(msg.pubKey)) {
-            visibleMessages.push(msg)
-          }
-          break
-        case MessageType.MODERATION:
-          const senderPk = msg.pubKey
-          const moderationType = msg.message.moderationType
-          const moderationTarget = msg.message.moderationTarget
-          if (channelOwner === senderPk && moderationType === 'ADD_MOD') {
-            channelModerators.push(moderationTarget)
-          } else if (channelOwner === senderPk && moderationType === 'REMOVE_MOD') {
-            const indexToRemove = channelModerators.findIndex(el => el === moderationTarget)
-            if (indexToRemove !== -1) {
-              channelModerators.splice(indexToRemove, 1)
+      const channelModerators = []
```

Comment:
Is our moderation / blocking code still in place? 

Is this worth carrying with us or should we remove it and address later?

Reply:
I think we should abandon this and reimplement later, If I remember correctly, that was bound to old zcash infrastructure.